ipaserver/install/server/upgrade.py

```diff
@@ -1863,11 +1863,15 @@ def upgrade_configuration():
         cainstance.ensure_ipa_authority_entry()
 
     set_sssd_domain_option('ipa_server_mode', 'True')
+    set_sssd_domain_option('ipa_server', api.env.host)
 
     sssdconfig = SSSDConfig.SSSDConfig()
     sssdconfig.import_config()
     sssd_enable_service(sssdconfig, 'ifp')
 
+    sssd = services.service('sssd', api)
+    sssd.restart()
```

Comment:
Why is the restart necessary?

Reply:
Yes, otherwise sssd wouldn't apply its configuration change.

Reply:
Thanks for the explanation